_posts/2020-09-16-hibernate-search.md

```diff
@@ -0,0 +1,556 @@
+---
+title: "Full-Text Search with Hibernate Search and Spring Boot"
```

Comment:
We'll have to think of a better title because there is no Spring Boot in the article.